packages/expo-module-scripts/eslint.config.base.js

```diff
@@ -0,0 +1,58 @@
+const universeNativeConfig = require('eslint-config-universe/flat/native');
+const universeNodeConfig = require('eslint-config-universe/flat/node');
+
+const universeRoot = require.resolve('eslint-config-universe/package.json');
+const { defineConfig } = require(
+  require.resolve('eslint/config', {
+    paths: [universeRoot],
+  })
+);
+const globals = require(require.resolve('globals', { paths: [universeRoot] }));
```

Comment:
it's not hurt to add `eslint` and `globals`  as devDeps / peerDeps to expo-module-scripts. in the meantime i just don't want to introduce new deps. this pr tries to reuse the deps from eslint-config-universe.

open to change though

Reply:
Personally, I don't mind, since it's a bit overkill to re-expose this sub-dependency here or to extend the config in such a way that this carries over, so `require.resolve` is the most straightforward solution :+1: 

nit: Might be worth adding a comment here though with the explanation or a todo for the future